src/Type/Enum/ScriptLoadingGroupEnum.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Register the ScriptLoadingGroupEnum Type to the Schema
+ *
+ * @package WPGraphQL\Type\Enum
+ * @since TBD
+ */
+
+namespace WPGraphQL\Type\Enum;
+
+/**
+ * Class ScriptLoadingGroupEnum
+ */
+class ScriptLoadingGroupEnum {
+
+	/**
+	 * Register the ScriptLoadingStrategy Enum Type to the Schema
+	 *
+	 * @return void
+	 */
+	public static function register_type() {
+		register_graphql_enum_type(
+			'ScriptLoadingGroupEnum',
```

Comment:
Change the name to `ScriptLoadingLocationEnum`

Reply:
Fixed in https://github.com/wp-graphql/wp-graphql/pull/3196/commits/cd6c302b74dacfa6d7175d96091b3740d71cea77